ui/src/views/404/CTA/style.css

```diff
@@ -0,0 +1,33 @@
+CTA {
```

Comment:
ideally, these should be classes at top level. need to add to linter though, so only if you want to here

Reply:
Do you mean we should follow the CSS class selector convention, like:

```css
.cta {
  background-color: hotpink;
}
```

Because "selectors" don't really correlate with actual HTML classes, is it necessary to preserve the illusion that they do?

Reply:
not necessary no, not a blocker. i was going to lint for classes at top level after using css modules, it feels (to me) really natural to write `.component1` and `style={styles.component1}` but it's really on style

Reply:
expect _maybe_ that's clearer you've invented them and they're names, rather than appearing to be custom elements?

Reply:
I see, I take your point. Early on, Zef was showing me some CSS he'd written:

```css
body {
  background-color: var(--colour-brand-blue);
}
```

For a short time while Zef was explaining his choices, I was certain that this was updating the background colour of the actual `<body>`. Of course it wasn't, it was being applied to `<div styles={body}>` in the JSX, but I can see this approach becoming confusing.

Using `.classname` convention might provide a better signifier as to the intention of these rules; that they are supposed to be applied to a particular element, like classes.